gallery/src/components/NxRadio/NxRadioDisabledExample.tsx

```diff
@@ -12,25 +12,23 @@ export default function NxRadioDisabledExample() {
   const [city, setCity] = useState<string | null>('fulton');
 
   return (
-    <>
```

Comment:
Changes to this file are now extraneous to the purpose of this PR, please undo them to keep things relevant